go/engine/engine.go

```diff
@@ -29,16 +30,30 @@ type UIDelegateWanter interface {
 func runPrereqs(e Engine, ctx *Context) (err error) {
 	prq := e.Prereqs()
 
+	// in standalone mode, try logging in if necessary:
+	/*
+		if e.G().Env.GetStandalone() && (prq.Session || prq.Device) {
+			e.G().Log.Debug("standalone mode + session prereq = %v, device prereq = %v: attempting autologin", prq.Session, prq.Device)
+			eng := newLoginProvisionedDevice(e.G(), "")
+			if err = RunEngine(eng, ctx); err != nil {
+				e.G().Log.Debug("standalone mode autologin failed: %s", err)
+				return err
+			}
+		}
+	*/
+
 	if prq.Session {
 		var ok bool
 		ok, _, err = IsLoggedIn(e, ctx)
 		if !ok {
-			urlError, isURLError := err.(*url.Error)
-			context := ""
-			if isURLError {
-				context = fmt.Sprintf("Encountered a network error: %s", urlError.Err)
+			if serr := tryStandaloneLogin(e, ctx); serr != nil {
+				urlError, isURLError := err.(*url.Error)
```

Comment:
right, but it seems OK to make this same translation in either call, right?  it would be weird if in one case you need to translate the errors and in the other case you don't.